homeassistant/components/jewish_calendar/services.yaml

```diff
@@ -0,0 +1,29 @@
+count_omer:
+  fields:
+    date:
+      required: true
+      example: "2025-04-14"
+      selector:
+        date:
+    nusach:
+      example: "sfarad"
+      default: "sfarad"
+      selector:
+        select:
+          translation_key: "nusach"
+          options:
+            - "sfarad"
+            - "ashkenaz"
+            - "adot_mizrah"
+            - "italian"
+    language:
+      required: true
+      default: "he"
+      example: "he"
+      selector:
+        select:
+          translation_key: "language"
+          options:
+            - "en"
+            - "he"
+            - "fr"
```

Comment:
Okay so I would expect a language selector to work here, but I don't find any examples of it being used, so you might have to play around a bit

Reply:
What do you mean play around a bit?

Reply:
I know there is a `homeassistant.helpers.selector.LanguageSelector`, but I don't 100% know how to use it in a services.yaml, but it should be straight forward.

https://www.home-assistant.io/docs/blueprint/selectors/#language-selector

Reply:
Ok, looks like it's working nicely with the language selector.